int32 id
int32 set_id
---
bool idgrab
